src/modules/stats/sagas.ts

```diff
@@ -8,13 +9,13 @@ import {
 } from './action'
 import { WeeklyStats } from './types'
 
-export function* statsSaga(builder: BuilderAPI) {
+export function* statsSaga(_builder: BuilderAPI) {
```

Comment:
I think that we should remove this if we're not using the builder API here, what do you think?